src/test/java/io/lettuce/core/pubsub/PubSubReactiveTest.java

```diff
@@ -343,6 +345,13 @@ void unsubscribe() throws Exception {
 
     }
 
+    @Test
+    void ssubscribe() throws Exception {
+        StepVerifier.create(pubsub.ssubscribe(channel)).verifyComplete();
```

Comment:
Have in mind that here we are testing in a non-cluster environment.

I think we still need to test that, because it is a valid (albeit - probably never used in real life) scenario.

Reply:
this will also serve for Redis Enterprise.